src/components/Portfolio/LoanFilterBar.vue

```diff
@@ -83,7 +84,8 @@ const handleExportClick = async () => {
 
 		const userId = data?.my?.userAccount?.id;
 		if (!userId) {
-			console.error('No user ID available for export');
+			logFormatter('No user ID available for export');
+			this.$showTipMsg('There was a problem preparing your loan export.', 'error');
```

Comment:
`this.$showTipMsg` won't exist for components that use composition API. If we want to show a tip message, that functionality will need to be made available and usable by the composition API. See how we `provide` different functionality in `main.js`. Might be a good conversation for the business owner/stakeholder for this migration. Or maybe we just match what we used to do in the monolith, did we show a notification on failure? If not, we can just use `logFormatter` and leave it there, since these errors shouldn't happen.